modules/cluster-logging-clo-status-comp.adoc

```diff
@@ -62,12 +73,15 @@ kibana-5bd5544f87                         1         1         1       157m
 
 .. Get the status of the ReplicaSet:
 +
+[source,terminal]
 ----
 $ oc describe replicaset cluster-logging-operator-574b8987df
 ----
 +
 The output includes the following status information:
```

Comment:
@mburke5678, from what I've seen, we've been omitting _The output includes the following status information:_ or similar in favor of `.Example output`.